docs/release-notes/2.24.0.md

```diff
@@ -0,0 +1,52 @@
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8027](https://jira.percona.com/browse/PMM-8027): Integrated Alerting: WebHook Notification Channels
+- [PMM-8301](https://jira.percona.com/browse/PMM-8301): Add support data collection for the new dimensions of pg_stat_monitor in QAN
+- [PMM-8588](https://jira.percona.com/browse/PMM-8588): PostgreSQL Histograms in QAN with pg_stat_monitor
+- [PMM-8632](https://jira.percona.com/browse/PMM-8632): Query command Type for PostgreSQL instances with pg_stat_monitor
+
+## Improvements
+
+- [PMM-8803](https://jira.percona.com/browse/PMM-8803): User friendly notification if you have incompatible util to service
+- [PMM-8636](https://jira.percona.com/browse/PMM-8636): Alerting: Improve alert message by adding more context to it
+- [PMM-8122](https://jira.percona.com/browse/PMM-8122): Alerting: UI does not indicate a port is needed in configuration for SMTP communication channel
+- [PMM-8484](https://jira.percona.com/browse/PMM-8484): PostgreSQL 14 support
+- [PMM-7297](https://jira.percona.com/browse/PMM-7297): update vertamedia-clickhouse-datasource plugin
+
+## Bugs Fixed
+
+- [PMM-8975](https://jira.percona.com/browse/PMM-8975): Scheduled backups - cron expression incorrectly parsed
+- [PMM-8387](https://jira.percona.com/browse/PMM-8387): MySQL InnoDB Details dashboard is not in the left menu
+- [PMM-8858](https://jira.percona.com/browse/PMM-8858): Dashboards: No Host uptime on Homepage for RDS instances 
+- [PMM-8611](https://jira.percona.com/browse/PMM-8611): Agents state shows DOWN if the respective metric is unknown
+- [PMM-8393](https://jira.percona.com/browse/PMM-8393): Alert rules not executed after upgrading PMM Server running as Docker container
+- [PMM-8058](https://jira.percona.com/browse/PMM-8058): IA: Firing alerts disappear after docker restart 
+- [PMM-8089](https://jira.percona.com/browse/PMM-8089): lost mongodb_memory metrics for mapped parameters
+- [PMM-9100](https://jira.percona.com/browse/PMM-9100): SHOW BINARY LOGS not parsed properly for MySQL8
+- [PMM-9125](https://jira.percona.com/browse/PMM-9125): VM Agent has higher memory usage and CPU usage post upgrading PMM Server to 2.24.0 RC
+- [PMM-8859](https://jira.percona.com/browse/PMM-8859): Increase in Memory for postgres_exporter and vmagent with lot of custom queries
+- [PMM-9108](https://jira.percona.com/browse/PMM-9108): Installed Grafana Plugins on PMM 2.23.0 are not available after upgrade to 2.24.0
+- [PMM-9046](https://jira.percona.com/browse/PMM-9046): Incorrect link to instructions about enabling
+- [PMM-8952](https://jira.percona.com/browse/PMM-8952): QAN: No table/indexes found with pg_stat_monitor 
```

Comment:
```suggestion
- [PMM-8952](https://jira.percona.com/browse/PMM-8952): QAN: No table/indexes tab when using pg_stat_monitor 
```

Reply:
I've renamed the issue to 
"Query Analytics: No table/indexes information for Views when PostgreSQL server monitored with pg_stat_monitor" 
this is what is the problem about. for Views only